app/model/schema/messaging.py

```diff
@@ -16,9 +16,45 @@
 
 SPDX-License-Identifier: Apache-2.0
 """
+import re
+import string
 from typing import Optional
 
-from pydantic import BaseModel, EmailStr, Field, Json, conlist, constr
+from pydantic import (
+    BaseModel,
+    EmailStr,
+    Field,
+    Json,
+    conbytes,
+    conlist,
+    constr,
+    root_validator,
+    validator,
+)
+
+############################
+# COMMON
+############################
+
+
+def _get_unprintable_ascii_chars() -> list[str]:
+    return [chr(c) for c in range(128) if chr(c) not in string.printable]
+
+
+# ASCII control characters
+UNPRINTABLE_ASCII_CHARS = tuple(_get_unprintable_ascii_chars())
+# Not allowed characters for path
+INVALID_PATH_CHARS = "".join(UNPRINTABLE_ASCII_CHARS)
+# Not allowed characters for file name
+INVALID_FILENAME_CHARS = INVALID_PATH_CHARS + "/"
+# Not allowed characters for path on Windows OS
+INVALID_WIN_PATH_CHARS = INVALID_PATH_CHARS + ':*?"<>|\t\n\r\x0b\x0c'
+# Not allowed characters for file name on Windows OS
+INVALID_WIN_FILENAME_CHARS = INVALID_FILENAME_CHARS + INVALID_WIN_PATH_CHARS + "\\"
+# Regex object for file name validation
+RE_INVALID_WIN_FILENAME = re.compile(
+    f"[{re.escape(INVALID_WIN_FILENAME_CHARS):s}]", re.UNICODE
+)
```

Comment:
I added comments for const vars which are used for file name validation.
(Inspired by https://github.com/thombashi/pathvalidate repository and https://github.com/thombashi/pathvalidate/blob/master/pathvalidate/_base.py#L17-L23 line)